Resources/public/css/alloyeditor/general.css

```diff
@@ -3,12 +3,49 @@
  * For full copyright and license information view LICENSE file distributed with this source code.
  */
 
+.ez-view-richtexteditview .ez-richtext-editor {
+    padding: 0 .5em;
+}
+
+.ez-view-richtexteditview .ez-richtext-editor h1.is-block-focused,
```

Comment:
Isn't possible to add the spacing without being focused, just `.ez-view-richtexteditview .ez-richtext-editor h1`?
If we add space when focused, would (probably) generate another usability issue, because text doesn't have to move, the toolbar has to adapt to the context.
Besides of this, Isn't possible to add the spacing after/below the paragraph instead of on top of it?